src/algorithms.jl

```diff
@@ -45,6 +45,20 @@ and `f'` is the derivative of `f` (gradient in the multivariate case).
 """
 struct MeanValueEnclosure <: AbstractDirectRangeAlgorithm end
 
+
+"""
+    AffineArithmeticEnclosure <: AbstractDirectRangeAlgorithm
+
+Data type to bound the range of `f` over `X` using affine arithmetic.
+See [`AffineArithmetic.jl`](https://github.com/JuliaIntervals/AffineArithmetic.jl) for more details.
+
+### Notes
+
+To use this algorithm, you need to load `AffineArithmetic.jl`, note also that `AffineArithmetic.jl`
+currently supports only arithmetic operations.
```

Comment:
As opposed to?

Reply:
Everything else? I mean AA.jl currently only has `+, -, *, /, ^`

Reply:
Maybe it is not clear to a reader what else you would want. But I can see that any example would be arbitrary.

Reply:
Yeah I basically wanted to give a heads up that no elementary function is supported